pkg/kotsadm/api_objects.go

```diff
@@ -210,7 +212,8 @@ func apiDeployment(deployOptions types.DeployOptions) *appsv1.Deployment {
 							Ports: []corev1.ContainerPort{
 								{
 									Name:          "http",
-									ContainerPort: 3000,
+									ContainerPort: hostnetwork.ContainerPorts(deployOptions.UseHostNetwork).KotsadmKotsadmAPI,
```

Comment:
Will k8s start a pod if it requests a containerport and CNI isn't running? If it will, should we just have that be unconditional?